app/graphql/types/query_type.rb

```diff
@@ -118,7 +118,7 @@ def waste_location_type(id:)
     def news_items_data_providers(category_id: nil)
       return DataProvider.joins(:news_items).order(:name).uniq if category_id.blank?
 
-      NewsItem.with_category(category_id).map(&:data_provider).sort_by do |data_provider|
+      NewsItem.with_category(category_id).includes(:data_provider).map(&:data_provider).sort_by do |data_provider|
```

Comment:
do you mind line breakings here to satisfy linter?

<img width="819" alt="Bildschirmfoto 2022-01-25 um 13 00 57" src="https://user-images.githubusercontent.com/1942953/150973365-db77bef2-6cde-411b-9824-99eb75006df4.png">

Reply:
Solved in https://github.com/ikuseiGmbH/smart-village-app-mainserver/commit/444c7559fc08f7ab7a3cb7d777551753d812ea5b